setup.py

```diff
@@ -34,6 +34,7 @@
                'scripts/targetcli',
                'daemon/targetclid'
               ],
+    data_files = [('lib/systemd/system', ['systemd/targetclid.socket', 'systemd/targetclid.service'])],
```

Comment:
@maurizio-lombardi 

Please fix the path.

it should be:
```
 data_files = [('/lib/systemd/system', ['systemd/targetclid.socket', 'systemd/targetclid.service'])],
```

Reply:
Thanks, done. 
For some reasons it worked when I tested it on RHEL8.

Reply:
`directory` path  in `data_files` must be relative !. Absolute path is deprecated and already broken on Arch, Gentoo , Fedora and RedHat > 8 . [Docs](https://docs.python.org/3/distutils/setupscript.html#installing-additional-files)